### DISCUSSION TITLE
save_audio: saving the timestamps as part of the filename?

### DISCUSSION THREAD

**Discussion by dgoryeo** [snakers4/silero-vad#299] [Q&A]
I'd like to be able to save the 'start', 'end' timestamps as part of the filename when saving each chunk for post-processing. I do not know how to do it at the moment --I'm sure it is possible by extracting the values from the output and to add it to the path before passing to save_audio. But I do not know how to do that. I appreciate any pointer on how it can be done. 

Also, I wonder if it would be a useful feature to have in future? 

Thanks

**Comment by snakers4** [comment]
Just extract the timestamps, read the audio array, and just slice the array and save each chunk manually.

**Comment by dgoryeo** [comment]
Here is the code I am using to save each chunk. But I get syntax error on collect_chunks --have tried various permutations but can't work it out:

```
speech_timestamps = get_speech_timestamps(wav, model)
for i in speech_timestamps:
    save_audio(
    str(i) + "_" + str(i['start']) +"_" + ".wav",
    collect_chunks(i, wav),
    sampling_rate=16000,
    )

```

What am I doing wrong?

Thanks
